src/util/serialize.cpp

```diff
@@ -354,6 +354,55 @@ std::string deSerializeJsonString(std::istream &is)
 	return os.str();
 }
 
+std::string serializeJsonStringIfNeeded(const std::string &s)
```

Comment:
out of scope. This is a util function that wraps around other functions, many other places used this and it should be done separately